walkthroughs/week-5-tdd/project/src/main/java/com/google/sps/FindMeetingQuery.java

```diff
@@ -54,15 +54,31 @@
     }
 
     Collection<TimeRange> occupiedTimeRanges =
-        getRelevantTimeRangesFromEvents(events, attendees);
+        getConcernedAttendeesTimeRangesFromEvents(events, attendees);
 
     return getAvailabeTimeRanges(occupiedTimeRanges, requestedDuration);
   }
 
+  /**
+  * Finds the existing {@code TimeRange} that correspond to {@code Event} having
+  * overlapping attendees with a set of attendees that we are concerned with.
```

Comment:
nit: use param names when available (i.e. {@code events}, {@code attendees})

Reply:
I would also change this to be more about "what" you're doing rather than "how" (i.e. instead of "overlapping attendees", maybe something like "timeranges of events that include the given attendees")